src/app/modules/project/overview/overview.component.ts

```diff
@@ -53,30 +61,49 @@ export class OverviewComponent implements OnInit {
 
   private searchSubject = new BehaviorSubject<InternalSearchQuery>(null);
 
+  /**
```

Comment:
Component does not follow the order defined the coding guidelines: https://github.com/DigitalExcellence/dex-frontend/wiki/Coding-Guidelines#-methods-and-variables-order